site/docs/usage/self-hosting.md

```diff
@@ -23,9 +23,7 @@ promptfoo provides a Docker image that allows you to host a central server that
 
 The self-hosted app consists of:
 
-- Next.js application that runs the web UI.
-- Filesystem store that persists the eval results.
-- Key-value (KV) store that persists shared data (redis, filesystem, or memory).
+- Express server that serves the web UI and API.
```

Comment:
Just make this

> The self-hosted app is an Express server that serves the web UI and API.

(sorry can't make the inline suggestion on a non-edited line)